sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -122,16 +131,18 @@ public KeyVaultKeyStore() {
         keyVaultCertificates = new KeyVaultCertificates(refreshInterval, keyVaultClient);
         classpathCertificates = new ClasspathCertificates();
         jreCertificates = JreCertificates.getInstance();
-        allCertificates = Arrays.asList(keyVaultCertificates, classpathCertificates, jreCertificates);
+        customCertificates = new FileSystemCertificates(customPath);
+        wellKnowCertificates = new FileSystemCertificates(wellKnowPath);
+        allCertificates = Arrays.asList(keyVaultCertificates, classpathCertificates, jreCertificates, wellKnowCertificates, customCertificates);
```

Comment:
1. sort by priority: jreCertificates -> wellKnowCertificates -> customCertificates -> keyVaultCertificates -> classpathCertificates.
2. Add warn log if alias duplicated.
3. Add unit test for it.